### DISCUSSION TITLE
no sound and audio after the last upgrade

### DISCUSSION THREAD

**Discussion by constantius24** [mocaccinoOS/mocaccino#150] [General]
there iso no pulseaudio package? Only pulseaudio-qt? I have enabled pipewire but nothing helps. Pavucontrol cannot connect to pulseaudui service

**Comment by constantius24** [comment]
Kernel 6.6.63
GNOME Shell 45.5

**Comment by constantius24** [comment]
gnome settings - no sound device recognized

**Comment by constantius24** [comment]
systemctl --user status pipewire
● pipewire.service - PipeWire Multimedia Service
     Loaded: loaded (/usr/lib/systemd/user/pipewire.service; enabled; preset: e>
     Active: active (running) since Wed 2024-12-18 19:10:19 CET; 17min ago
 Invocation: bcf848b2a2ae4fe18cd01a8ced164f3c
TriggeredBy: ● pipewire.socket
   Main PID: 1502 (pipewire)
      Tasks: 3 (limit: 19018)
     Memory: 3.6M (peak: 3.9M)
        CPU: 49ms
     CGroup: /user.slice/user-1000.slice/user@1000.service/session.slice/pipewi>
             └─1502 /usr/bin/pipewire

**Comment by joostruis** [comment]
Make sure you have the package utils/yip-systemd installed this properly sets certain profiles on the system.

**Reply by constantius24** [reply]
yes its installed

**Comment by constantius24** [comment]
Did you switched from pulseaudio to pipewire? The package pulseaudio not exist?

**Comment by constantius24** [comment]
Can anybody help how to restore audio?

**Comment by msdobrescu** [comment]
Did you try this solution here, https://github.com/mocaccinoOS/mocaccino/discussions/141#discussioncomment-10714777 ?
Also, I remember this discussion: https://github.com/mocaccinoOS/mocaccino/discussions/135
Then, we switched to pipewire already. Check https://pipewire.pages.freedesktop.org/wireplumber/daemon/configuration/migration.html.
